.gitignore

```diff
@@ -167,3 +167,7 @@ cython_debug/
 
 # serialized files
 *.pkl
+
+# Windows venv folder/files
+Scripts
+pyvenv.cfg
```

Comment:
@KostiantynMoskalenko  Please fix empty new lines at the enf of file. Each file should have a new line at the end (you can also configure it in VScode to add new line of file save)

Reply:
What do you mean "new line"? What is it? What should I do? As far as I can see every file already has empty line in the end? BTW is it requirement of PEP8? Or something another?

Reply:
new line at the end of file

Reply:
it's already has because I've added it during conflicts resolution